doc/man/wesnoth.6

```diff
@@ -138,6 +138,9 @@ as explained below. Only these additional options can follow
 runs the game without any delays for graphic benchmarking. This is automatically enabled by
 .BR --nogui .
 .TP
+.B --noaddons
```

Comment:
This is one of the commits that will be rebased out. It's only here because I forgot to reset after cherry-picking it to make sure that it actually worked as intended.